docs/geedocs/5.2.5/answer/176738.html

```diff
@@ -256,7 +256,8 @@ <h2 id="cache_configuration">Cache configuration</h2>
 The asset cache size is controlled by <code>AssetCacheSize</code> and the asset version cache size is controlled by <code>VersionCacheSize</code>.
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
 
-Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>.
+Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>
+<p>Also included within <code>misc.xml</code> is an experimental option, which allows the user to limit the caches by memory usage instead of number of objects. This option is controlled by <code>LimitMemoryUtilization</code> and its default value is false. The maximum memory usage for each cache can be set with <code>MaxAssetCacheMemorySize</code> and <code>MaxVersionCacheMemorySize</code>. This size is measured in bytes and the default setting is 1GB. While this option is enabled, the caches will not be limited by the number of objects in the cache.</p>
```

Comment:
This documentation really needs to go into the 5.3.1 version. I haven't figured out how to handle that yet.